test/benchmarks/benchmark_compare256.cc

```diff
@@ -74,6 +74,9 @@ BENCHMARK_COMPARE256(unaligned_64, compare256_unaligned_64, 1);
 #endif
 #endif
 
+#ifdef X86_SSE2
+BENCHMARK_COMPARE256(unaligned_sse2, compare256_unaligned_sse2, x86_cpu_has_sse2);
+#endif
 #ifdef X86_SSE42_CMP_STR
 BENCHMARK_COMPARE256(unaligned_sse4, compare256_unaligned_sse4, x86_cpu_has_sse42);
 #endif
```

Comment:
Maybe it should be removed?
``compare256_unaligned_sse4`` function in deleted file ``arch/x86/compare256_sse42.c``.

Reply:
@KungFuJesus, is this the correct code? You have not deleted it.

Reply:
Sorry, got it now.